### ISSUE TITLE
Hopeing you could add support for teensy LC 

### ISSUE THREAD

**Issue by KennethMcNutt** [vtl/volvo-cem-cracker#21]
To use the MCP2551 transceiver with a Teensy LC microcontroller, you can follow these steps:

Step 1: Wiring
Connect the MCP2551 to your Teensy LC as follows:
- VCC to 5V on Teensy LC.
- GND to GND on Teensy LC.
- TXD to SPI MOSI (pin 11 on Teensy LC).
- RXD to SPI MISO (pin 12 on Teensy LC).
- CS to a digital pin (e.g., pin 10) on Teensy LC.
- INT (if used) connected accordingly.

Step 2: Install Libraries
In your Arduino IDE, install the "MCP_CAN_lib" by visiting "Sketch -> Include Library -> Manage Libraries", then search for and install the library.

Step 3: Initialize MCP2515
You'll require code similar to this in your setup function:
```cpp
#include https://github.com/Longan-Labs/Arduino_CAN_BUS_MCP2515
#include Adafruit MCP2515](https://reference.arduino.cc/reference/en/libraries/adafruit-mcp2515/
#include https://github.com/autowp/arduino-mcp2515

MCP2515 mcp2515(10); // Assuming CS pin is connected to digital pin 10

void setup() {
    Serial.begin(115200);
    SPI.begin();
    mcp2515.reset();
Step 4: Setting Bitrate
To set the CAN bit rate to 500 kbps, you can use the `setBitrate` function. For example:
```cpp
    mcp2515.setBitrate(CAN_500KBPS);
```

Step 5: Sending CAN Message
Here's an example of how you can send a CAN message using MCP2515 from your loop or main function:
```cpp
void loop() {
    // Prepare message
    uint8_t data[8] = {0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08};
    CAN_message_t msg;
    
    msg.id = /* Set the ID of your message */;
    msg.len = sizeof(data);
    memcpy(msg.buf, data, sizeof(data));

    // Send the prepared message using MCP2515
    mcp2515.sendMessage(&msg);

    delay(100); // Add a delay if necessary

}
```

In summary:
- Connect the MCP2551 transceiver to your Teensy LC.
- Install the "MCP_CAN_lib" library in Arduino IDE.
- Initialize and configure the MCP2515
I'm not sure if this will work.. 
and you may know an easier way than using two IC's ?

**Comment by vtl** [comment]
Teensy's built-in CAN is better and cheaper.
